src/users/dto/create-user.dto.ts

```diff
@@ -0,0 +1,12 @@
+import { IsEmail, IsNotEmpty, IsString } from 'class-validator';
+
+export class CreateUserDto {
+  @IsEmail()
+  email: string;
+
+  @IsString()
```

Comment:
**praise**: Combining `@IsString()` and `@IsNotEmpty()` for `password` ensures it is a non-empty string, enhancing security.